components/MessageContextMenuWrappers/MessageContextMenuWrapper.tsx

```diff
@@ -73,6 +76,17 @@ export const MessageContextMenuWrapper: FC<MessageContextMenuWrapperProps> = ({
     closeMenu();
   }, [message.content, message.contentFallback, closeMenu]);
 
+  const isFrame = isFrameMessage(message);
```

Comment:
this will be recomputed on every render
I think you can move the isFrameMessage method inside the frameURL useMemo and just test if frameURL is not null line 104 since frameURL will always be null if isFrame is false